universal/include/userver/formats/universal/universal.hpp

```diff
@@ -0,0 +1,235 @@
+#pragma once
+#include <userver/utils/constexpr_string.hpp>
+#include <userver/formats/parse/to.hpp>
+#include <userver/formats/serialize/to.hpp>
+#include <userver/utils/meta.hpp>
+#include <type_traits>
+#include <boost/pfr.hpp>
+
+
+USERVER_NAMESPACE_BEGIN
+
+namespace formats::universal {
+
+
+template <typename T>
+struct FieldConfig {
```

Comment:
Is there a test that putting a type with non-universal `Parse-Serialize` inside a struct is OK?